### PR TITLE
Dts 27186 large chart view

### PR DESCRIPTION
**Pull request by brahmanand1**

### CODE REVIEW

File: UI/src/app/component/multiline/multiline.component.ts [hunk 1]
```diff
@@ -52,27 +52,37 @@ export class MultilineComponent implements OnChanges {
...
+        d3.select(this.elem).select('.bstimeslider').remove();
           this.draw('update');
-        } else {
-          this.draw('new');
-        }
-      }
+      //   } else {
+      //     this.draw('new');
+      //   }
+      // }
```
Comment (BhagyashreeShettigar) [inline UI/src/app/component/multiline/multiline.component.ts:85]: remove commented code

File: UI/src/app/component/table/table.component.css [hunk 2]
```diff
@@ -40,7 +40,7 @@
 }
 
 :host::ng-deep tr td:first-child span{
-    position: relative;
+    /* position: relative; */
```
Comment (BhagyashreeShettigar) [inline UI/src/app/component/table/table.component.css:43]: remove commented code

Comment (brahmanand1) [reply UI/src/app/component/table/table.component.css]: removed

File: UI/src/app/component/multiline/multiline.component.ts (hunk 1 above)
Comment (brahmanand1) [reply UI/src/app/component/multiline/multiline.component.ts]: removed

**Review by BhagyashreeShettigar — APPROVED** [review]

**Review by sanshriv — APPROVED** [review]

**Pull request merged**